TLM/TLM/LoadingExtension.cs

```diff
@@ -146,8 +146,7 @@ public class LoadingExtension : LoadingExtensionBase {
                 Log.Info("Removing Controls from UI.");
                 if (ModUI.Instance != null) {
                     ModUI.Instance.CloseMainMenu(); // Hide the UI ASAP
```

Comment:
Would `.CloseMainMenu()` not be better located inside `.Destroy()` ? I assume the menu will need closing regardless of what's destroying the `ModUI.Instance`.

Should the `Log._Debug()` also move in to `.Destroy()`?

Reply:
The part you commented on is in the RebuildMenu(). Here it seems the idea is only to rebuild the parts that ModUI references. Destroy() rather is to destroy everything. So I m not sure I can follow you. Also same applies here, I only wanted to remove Finalizers, and try to keep all other changes at 0 if possible while keeping the same behaviour as before.